lib/geo_faker/multi_polygon.rb

```diff
@@ -0,0 +1,45 @@
+module GeoFaker
+  class MultiPolygon
+    attr_reader :coordinates
+
+    def initialize(coordinates)
+      @coordinates = coordinates
+    end
+
+    def self.from_geojson(geojson)
+      case geojson.fetch('type')
+      when 'MultiPolygon'
+        MultiPolygon.new(geojson.fetch('coordinates'))
+      when 'Polygon'
+        MultiPolygon.new([geojson.fetch('coordinates')])
+      end
+    end
+
+    def contains_point(point)
+      coordinates.any? { |polygon| point_in_poly(polygon[0], point) }
```

Comment:
Why `polygon[0]`? Can a polygon have more than one sub-polygon?

Reply:
Yes. In GeoJSON each Polygon is a list of rings (i.e. lon,lat [sic] coordinates where the first and last are the same). The first ring is the outer boundary, and the following rings are inner boundaries, i.e. holes.

As you can see in the test, we do not support holes yet, this would be the next thing to tackle and fairly easy too I think. I think we can just throw the edges from the hole boundaries into the list of outer edges and the odd/even intersection algorithm should still work. Just needs some refactoring to correctly extract the edges from multiple rings.

Nice article about GeoJSON: https://macwright.org/2015/03/23/geojson-second-bite.html#position